mysql/errname.go

```diff
@@ -275,7 +275,7 @@ var MySQLErrName = map[uint16]string{
 	ErrZlibZMem:                                 "ZLIB: Not enough memory",
 	ErrZlibZBuf:                                 "ZLIB: Not enough room in the output buffer (probably, length of uncompressed data was corrupted)",
 	ErrZlibZData:                                "ZLIB: Input data corrupted",
-	ErrCutValueGroupConcat:                      "Row %d was cut by GROUPCONCAT()",
+	ErrCutValueGroupConcat:                      "Some rows was cut by GROUPCONCAT()",
```

Comment:
Why change this?
```
mysql> show warnings;
+---------+------+---------------------------------+
| Level   | Code | Message                         |
+---------+------+---------------------------------+
| Warning | 1260 | Row 2 was cut by GROUP_CONCAT() |
+---------+------+---------------------------------+
1 row in set (0.00 sec)
```
If there is no row number, or we can use `GenWithStackByArgs`.

Reply:
we are can not got accurate row number, so it seems have no good value to set `GenWithStackByArgs` parameter ...and got stranger message like this~  :joy_cat: 

```
mysql> show warnings;
+---------+------+---------------------------------+
| Level   | Code | Message                         |
+---------+------+---------------------------------+
| Warning | 1105 | Row %d was cut by GROUPCONCAT() |
| Warning | 1105 | Row %d was cut by GROUPCONCAT() |
+---------+------+---------------------------------+
```
so I think maybe change a new message...

Reply:
s/was/were/